src/App.tsx

```diff
@@ -1,13 +1,23 @@
-import Logo from "@/assets/logo.png";
-import HelloWorld from "@/components/HelloWorld/HelloWorld";
+import { useEffect, useState } from "react";
 
 import styles from "./App.module.css";
+import Login from "./components/Login/Login";
+import Products from "./components/Products/Products";
```

Comment:
Here you are importing Login and Products from components folder, so import it like this: 
import {Login, Products} from "@/components";

We will connect on this.

Reply:
addressed the comment.